app/javascript/components/contextual-guidance.js

```diff
@@ -1,6 +1,7 @@
 function ContextualGuidance ($module) {
   if (!$module) $module = document
   this.$module = $module
+  throw new Error('Testing raven')
```

Comment:
I'm guessing this is temporary :-).

Reply:
Ha yup. I thought I should test it properly before shouting about this PR.